lib/bloc/account/account_bloc.dart

```diff
@@ -74,34 +74,44 @@ class AccountBloc {
 
 
   Stream<Map<String, DownloadFileInfo>>  chainBootstrapProgress;
-  BreezUserModel _currentUser;  
+  BreezUserModel _currentUser;
+  BreezBridge _breezLib;
+  Device _device;
   bool _allowReconnect = true;
+  bool _startedLightning = false;
 
   AccountBloc(Stream<BreezUserModel> userProfileStream) {
       ServiceInjector injector = new ServiceInjector();    
-      BreezBridge breezLib = injector.breezBridge;  
+      _breezLib = injector.breezBridge;
       BreezServer server = injector.breezServer;
       Notifications notificationsService = injector.notifications;
-      Device device = injector.device;      
+      _device = injector.device;
 
       _accountController.add(AccountModel.initial());
       _paymentFilterController.add(PaymentFilterModel.initial());
       //listen streams      
-      _listenUserChanges(userProfileStream, breezLib);
-      _listenNewAddressRequests(breezLib);
-      _listenWithdrawalRequests(breezLib);  
-      _listenSentPayments(breezLib);
-      _listenFilterChanges(breezLib);
-      _listenAccountChanges(breezLib);
-      _listenPOSFundingRequests(server, breezLib);
-      _listenMempoolTransactions(device, notificationsService, breezLib);
-      _listenRoutingNodeConnectionChanges(breezLib);
-       breezLib.bootstrapAndStart();      
-       _refreshAccount(breezLib);
-       _listenConnectivityChanges(breezLib);   
-       _listenReconnects(breezLib);
-       _listenRefundableDeposits(breezLib, device);
-       _listenRefundBroadcasts(breezLib);
+      _listenUserChanges(userProfileStream, _breezLib);
+      _listenNewAddressRequests(_breezLib);
+      _listenWithdrawalRequests(_breezLib);
+      _listenSentPayments(_breezLib);
+      _listenFilterChanges(_breezLib);
+      _listenAccountChanges(_breezLib);
+      _listenPOSFundingRequests(server, _breezLib);
+      _listenMempoolTransactions(_device, notificationsService, _breezLib);
+      _listenRoutingNodeConnectionChanges(_breezLib);
+    }
+
+    void startLightning() {
```

Comment:
Suggest to make this method private so we will be sure that is called only from the account bloc